tools/image_processing/yolo/yolo_prediction.xml

```diff
@@ -0,0 +1,107 @@
+<tool id="yolo_predict" name="Perform YOLO image labeling" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="24.2">
+    <description>with ultralytics</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <expand macro="edam" />
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+      export YOLO_CONFIG_DIR='\$HOME/.config/ultralytics' &&
+
+      mkdir -p ./input_images ./runs ./models ./results && 
+      
+      #for $filename in $input_images:
+          ln -s '$filename' './input_images/${filename.element_identifier}' &&
+      #end for
+
+      cp '$class_name' './models/class_name.txt' &&
+
+      cp '$model' './models/model.pt' &&
+		    
+      python '$__tool_directory__/yolov8.py' 
+		    --test_path='./input_images' 
+		    --model_path='./models' 
+		    --model_name='model' 
+		    --run_dir='./runs' 
+		    --save_dir='./results'
+		    --image_size='$image_size'
+		    --mode='$mode'
+		    --foldername='overlaid_images'
+		    --class_names_file='$class_name'
+		    --num_classes=`wc -l < ./models/class_name.txt`
+		    --confidence='$confidence'
+		    --iou='$iou'
+		    --max_det='$max_det'  
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_images" type="data" format="jpg,png,tiff" multiple="true"  label="Input images"/>
```

Comment:
> > **2)** I think you should also add to the help section, that only 2-D images are supported. What about multi-channel images, like RGB images? If not supported, please explicitly mention this in the help section.
> 
>  […]
> * 2,3.  I couldn't find any docs explicitly says this, but it seems all yolo models are trained using jpeg.

The tool accepts `png` and `tiff` besides of `jpg`, but the tests only cover the `jpg` case. Can you maybe add a (small) test for `png` and `tiff`, just so that we can be sure that the wrapper does handle conversion properly?

Or, alternatively:
```suggestion
        <param name="input_images" type="data" format="jpg" multiple="true"  label="Input images"/>
```

Reply:
> it cannot be only jpg. it works with tiff, haven't test png.
will extra test really neccessary?

Sorry we found recently that many tools claim to support certain file types which they do not, which creates negative user experiences, this is why I'm being a bit nitpicky here, maybe.

> that require another zenodo collections of images, it won't be small because it's for training, and i would say minimum we need 10 images each format, because we need to split.

I understand the issue for training, but for prediction it should be fine to just add a tiny test file for TIFF, and another tiny test file for PNG, right?

Reply:
ok. i will add tests for prediction and remove png tiff from training.xml

Reply:
@kostrykin done. all suggested changes should be include, except for the additional section for augmentation. I would prefer leave it as-is for now..maybe until the code supports other tasks, such as `tracking` etc..
thanks a lot for the review